app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] as required your application should be able to ask for confirmation of email. where it's enabled in your user.rb file. you should enable this action in your user.rb file by adding the `: confirmable` keyword.

Reply:
Dear @alexander16108 the confirmable keyword is now added to the user model for an email confirmation.